src/test/java/org/jabref/model/entry/identifier/DOITest.java

```diff
@@ -86,15 +91,24 @@ public void acceptDoiPrefixInShortDoi() {
     @Test
     public void acceptURNPrefix() {
         assertEquals("10.123/456", new DOI("urn:10.123/456").getDOI());
-        assertEquals("10.123/456", new DOI("urn:doi:10.123/456").getDOI());
```

Comment:
Note to self: `urn:doi` is not a valid DOI: See https://en.wikipedia.org/wiki/Digital_object_identifier#Standardization